includes/admin/post-types/meta-boxes/fields/llms.class.meta.box.fields.php

```diff
@@ -72,15 +73,19 @@ public function output() {
 
 		?>
 		<li class="<?php echo implode( ' ', $wrapper_classes ); ?>"<?php echo $controller . $controller_value; ?>>
+		<?php if ( ! empty( $this->field['desc'] ) || ! empty( $this->field['label'] ) ) : ?>
 			<div class="description <?php echo $this->field['desc_class']; ?>">
+			<?php if ( ! empty( $this->field['label'] ) ) : ?>
 				<label for="<?php echo $this->field['id']; ?>"><?php echo $this->field['label']; ?></label>
+			<?php endif ?>
 				<?php echo $this->field['desc']; ?>
 				<?php
 				if ( isset( $this->field['required'] ) && $this->field['required'] ) :
 					?>
 					<em>(required)</em><?php endif; ?>
 			</div>
-			<?php
+		<?php endif; ?>
+		<?php
```

Comment:
```suggestion
		<?php endif; 
```

Reply:
Cannot accept, because of phpcs, which is the reason why I wrote it that way :D
gonna find another way.